packages/deployments/contracts/contracts/core/connext/helpers/RelayerProxyHub.sol

```diff
@@ -337,9 +323,7 @@ contract RelayerProxyHub is RelayerProxy {
         (IGnosisHubConnector.GnosisRootMessageData)
       );
       IGnosisHubConnector(hubConnectors[fromChain]).executeSignatures(data._data, data._signatures);
-    }
-
-    if (fromChain == 42161 || fromChain == 421613) {
+    } else if (fromChain == 42161 || fromChain == 421613) {
```

Comment:
should we move these chains into a `Constants` or `ChainIds` library?

Reply:
Done.